src/app/beer_garden/api/http/handlers/v1/command_publishing_blocklist.py

```diff
@@ -0,0 +1,119 @@
+        parameters:
+          - name: command_publishing_id
+            in: path
+            required: true
+            description: id of entry in command publishing block list
+            type: string
+        responses:
+          204:
+            description: Command has been successfully removed from block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklist'
+          404:
+            $ref: '#/definitions/404Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        blocked_command = CommandPublishingBlockList.objects.get(
+            id=command_publishing_id
+        )
+        _ = self.get_or_raise(Garden, SYSTEM_UPDATE, name=blocked_command.namespace)
+        command_publishing_blocklist_delete(blocked_command)
+
+        self.set_status(204)
+
+
+class CommandPublishingBlocklistAPI(AuthorizationHandler):
+    def get(self):
+        """
+        ---
+        summary: Retrieve list of commands in publishing block list
+        responses:
+          200:
+            description: list of commands in publishing block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklistListSchema'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        permitted_blocklist_entries = self.permissioned_queryset(
+            CommandPublishingBlockList, SYSTEM_READ
+        )
+        response = {
+            "command_publishing_blocklist": CommandPublishingBlocklistSchema(many=True)
+            .dump(permitted_blocklist_entries)
+            .data
+        }
+
+        self.write(response)
+
+    def post(self):
+        """
+        ---
+        summary: Add a list of commands to event publishing block list
+        parameters:
+          - name: CommandPublishingBlocklist
+            in: body
+            description: The system, namespace and command name
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklistListInputSchema'
+        consumes:
+          - application/json
+        responses:
+          201:
+            description: list of commands that have been added to publishing block list
+            schema:
+              $ref: '#/definitions/CommandPublishingBlocklistListSchema'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Command Block List
+        """
+        commands = self.schema_validated_body(CommandPublishingBlocklistListInputSchema)
+        checked_gardens = []
+        for command in commands["command_publishing_blocklist"]:
+            if command["namespace"] not in checked_gardens:
+                _ = self.get_or_raise(Garden, SYSTEM_UPDATE, name=command["namespace"])
```

Comment:
I didn't notice this until I was testing it again, but using `self.get_or_raise` here has an unwanted side effect in that it returns a 404 if the garden doesn't exist.  In this context, the response should be a 400 if the garden doesn't exist since the user supplied bad data.  404 would indicate that the `/api/v1/commandpublishingblocklist` endpoint doesn't exist, which isn't true.

There's a few ways that you can handle this, but the easiest is probably to just catch the `NotFound` exception and re-raise as a `BadRequest`.  Something like:

```python
# This goes up top with the imports
from beer_garden.api.http.exceptions import BadRequest, NotFound

# Then here you would do
try:
    _ = self.get_or_raise(Garden, SYSTEM_UPDATE, name=command["namespace"])
except NotFound:
   raise BadRequest(reason=f"Invalid garden name: {command['namespace']}")
```